engine/src/main/java/org/entando/entando/web/database/validator/DatabaseValidator.java

```diff
@@ -37,4 +42,20 @@ public void validate(Object o, Errors errors) {
         //nothing to do
     }
 
+    @Override
+    public boolean isValidField(String fieldName) {
+        List<String> fields = new ArrayList<>();
+        fields = getAllFields(fields, DumpReportDto.class);
+        return fields.contains(fieldName);
+    }
+
+    List<String> getAllFields(List<String> fields, Class<?> type) {
```

Comment:
This is identical to the one in the DataTypeValidator. Can we move to a utility class?